tests_integration/test_object.py

```diff
@@ -53,3 +53,32 @@ def test_object_table(runner, test_database, snowflake_session):
     assert (
         len(row_from_cursor(snowflake_session.execute_string(f"show tables")[-1])) == 0
     )
+
+
+@pytest.mark.integration
+def test_list_with_scope(runner, test_database, snowflake_session):
+    # create a table in a schema other than schema of the current connection
+    other_schema = "other_schema"
+    public_table = ObjectNameProvider(
+        "Test_Object_Table"
+    ).create_and_get_next_object_name()
+    other_table = object_name = ObjectNameProvider(
+        "Test_Object_Table"
+    ).create_and_get_next_object_name()
+    snowflake_session.execute_string(
+        f"use schema public; create table {public_table} (some_number NUMBER);"
+    )
+    snowflake_session.execute_string(
+        f"create schema {other_schema}; create table {other_table} (some_number NUMBER);"
+    )
+    result_list_public = runner.invoke_with_connection_json(
+        ["object", "list", "table", "--schema", "public"]
```

Comment:
shouldn't the command be `["object", "list", "table", "--in", "schema", "public"]`?